vault/core.go

```diff
@@ -517,9 +517,9 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 	// Wrap the physical backend in a cache layer if enabled
 	if !conf.DisableCache {
 		if txnOK {
-			c.physical = physical.NewTransactionalCache(phys, conf.CacheSize, conf.Logger)
+			c.physical = physical.NewTransactionalCache(phys, conf.CacheSize, nil, conf.Logger)
```

Comment:
Maybe we should just move these consts over so that we don't have to deal with ensuring this stays in sync. Thoughts?

Reply:
I was going to add the values here but the constants aren't in this repo.  We'll just have to sync them.